discovery_engine/lib/src/ffi/types/trending_topic.dart

```diff
@@ -15,58 +15,40 @@
 import 'dart:ffi' show Pointer;
 
 import 'package:equatable/equatable.dart';
-import 'package:xayn_discovery_engine/src/domain/models/embedding.dart';
 import 'package:xayn_discovery_engine/src/domain/models/trending_topic.dart';
-import 'package:xayn_discovery_engine/src/domain/models/unique_id.dart';
 import 'package:xayn_discovery_engine/src/ffi/genesis.ffigen.dart';
 import 'package:xayn_discovery_engine/src/ffi/load_lib.dart' show ffi;
-import 'package:xayn_discovery_engine/src/ffi/types/embedding.dart';
 import 'package:xayn_discovery_engine/src/ffi/types/string.dart' show StringFfi;
 import 'package:xayn_discovery_engine/src/ffi/types/uri.dart' show UriFfi;
-import 'package:xayn_discovery_engine/src/ffi/types/uuid.dart';
 
 class TrendingTopicFfi with EquatableMixin {
```

Comment:
I would also check if we use trending topics anywhere or do we still have plans to use them, maybe we could remove them? cc @LeifXayn, @frankpepermans

Reply:
good question. if we want to remove them, then it's probably better to do so in a follow up pr.

Reply:
yes yes, if we decide to remove it it should be done in a separate PR